unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ProfileHUD/ProfileHUDController.cs

```diff
@@ -0,0 +1,41 @@
+﻿using UnityEngine;
+using DCL.Interface;
+
+public class ProfileHUDController : IHUD
+{
+    private const string URL_CLAIM_NAME = "https://avatars.decentraland.org/claim";
+
+    internal ProfileHUDView view;
+
+    private UserProfile ownUserProfile => UserProfile.GetOwnUserProfile();
+
+    public ProfileHUDController()
+    {
+        view = UnityEngine.Object.Instantiate(Resources.Load<GameObject>("ProfileHUD")).GetComponent<ProfileHUDView>();
+        view.name = "_ProfileHUD";
+
+        view.buttonLogOut.onClick.AddListener(WebInterface.LogOut);
+        view.buttonClaimName.onClick.AddListener(()=> WebInterface.OpenURL(URL_CLAIM_NAME));
```

Comment:
I see that the redirection to `avatars.decentraland.org` is implemented in code, but I don't see the Claim Name button in the HUD.

Reply:
> I see that the redirection to `avatars.decentraland.org` is implemented in code, but I don't see the Claim Name button in the HUD.

If you already have a name claimed button won't be displayed

Reply:
Oh, ok! Make sense :)